ee/main.go

```diff
@@ -214,6 +214,7 @@ func main() {
 			PrimehubUrl:           primehubUrl,
 			EngineImage:           engineContainerImage,
 			EngineImagePullPolicy: engineContainerPullPolicy,
+			PhfsPVC:               viper.GetString("jobSubmission.phfsPVC"),
```

Comment:
We can move phfsPVC to upper.

And have a phfsPVC variable
```
if `viper.GetString("phfsPVC")` avaiable, use it.
if `viper.GetString("jobSubmission .phfsPVC")` avaiable, use it.
```

Reply:
main.go line 132.